cmd/buildah/mount.go

```diff
@@ -51,7 +51,7 @@ func mountCmd(c *cobra.Command, args []string, noTruncate bool) error {
 		// Differently, allow the mount if we are already in a userns, as the mount point will still
 		// be accessible once "buildah mount" exits.
 		if os.Getenv(startedInUserNS) != "" && store.GraphDriverName() != "vfs" {
-			return fmt.Errorf("cannot mount using driver %s in rootless mode", store.GraphDriverName())
+			return fmt.Errorf("cannot mount using driver %s in rootless mode.  You need to run it in a `buildah unshare` session", store.GraphDriverName())
```

Comment:
There's a double space between the sentences.

Reply:
I assumed we were using two spaces instead of one.  Fixed now